### ISSUE TITLE
UI/CSS Issue

### ISSUE THREAD

**Issue by OneNiNE87** [C0untFloyd/roop-unleashed#913]
**Describe the bug**
When the columns goes beyond the specified number, it creates row 2 and that takes the scroll bar away and the only way to scroll would be "TAB".
```
with gr.Column(scale=2):
                with gr.Row():
                    input_faces = gr.Gallery(label="Input faces gallery", allow_preview=False, preview=False, height=138, columns=64, object_fit="scale-down", interactive=False)

```
Tried playing with the CSS, checked recent commits and I was able to fix it but "component-6 (svelte-12cmxck) and "component-45 (svelte-au1olv) appeared out of nowhere.
```
mycss = """
        span {color: var(--block-info-text-color)}
        #fixedheight {
            max-height: 238.4px;
            overflow-y: auto !important;
        }
        .image-container.svelte-1l6wqyv {height: 100%}

    """
```
**To Reproduce**
1. Go to 'FaceSwap Tab'
2. Click on 'Upload 65+ Images'
3. Scroll down to 'Not Required'

**Details**
What OS are you using?
- [ ] Linux
- [ ] Linux in WSL
- [x] Windows 
- [ ] Mac

Are you using a GPU?
- [ ] No. CPU FTW
- [x] NVIDIA
- [ ] AMD
- [ ] Intel
- [ ] Mac

**Which version of roop unleashed are you using?**
16.9.2024 v4.2.8

**Screenshots**

**Comment by zappityzap** [comment]
Uploading more than columns=N images makes a second row of thumbnails in the gallery, and then the scroll bar disappears.

Removing `height=138, columns=64,` brings the scroll bar back and looks better to me, but I'm not sure what the intention is for that part of the UI.

**Comment by C0untFloyd** [comment]
> but I'm not sure what the intention is for that part of the UI.

Between old and new gradio versions there were several bugs and bugfixes where sometimes the height wasn't right or thumb images were displayed incorrectly, stuff like that. I could fix this back then by limiting the number of cols to 8. Some weeks ago there were reports about missing scrollbars when using more than 8 images. I then increased the column number to 64 but then the height was wrong so I also had to use that fixed height. That's where we are now. From your description it sounds like it's working again with the latest gradio but I'm really sick of it. Besides...IMO using more than 8 faces is a tad bit exaggerated anyway 😃
